doc/aria-supported.md

```diff
@@ -1,5 +1,11 @@
 # ARIA Roles and Attributes unsupported in axe-core.
 
+It can be difficult to know which features of web technologies are accessible across different platforms, and with different screen readers and other assistive technologies. Axe-core does some of this work for you, by raising issues when accessibility features are used that are known to cause problems.
+
+This page contains a list of ARIA 1.1 features that axe-core raises as unsupported. To learn more about accessibility support in our article [We’ve got your back with “Accessibility Supported” in axe](https://www.deque.com/blog/weve-got-your-back-with-accessibility-supported-in-axe/).
```

Comment:
This sentence isn't quite complete, but it could be easily fixed by removing the first word:
`Learn more about accessibility support in our article...`